android/src/main/java/com/tailscale/ipn/ui/view/MainView.kt

```diff
@@ -518,24 +525,70 @@ fun ConnectView(
 fun PeerList(
     viewModel: MainViewModel,
     onNavigateToPeerDetails: (Tailcfg.Node) -> Unit,
-    onSearchBarClick: () -> Unit
+    onSearchBarClick: () -> Unit,
+    onSearch: (String) -> Unit
 ) {
   val peerList by viewModel.peers.collectAsState(initial = emptyList<PeerSet>())
   val searchTermStr by viewModel.searchTerm.collectAsState(initial = "")
   val showNoResults =
       remember { derivedStateOf { searchTermStr.isNotEmpty() && peerList.isEmpty() } }.value
 
   val netmap = viewModel.netmap.collectAsState()
+  val focusManager = LocalFocusManager.current
+  var isSearchFocussed by remember { mutableStateOf(false) }
   var isListFocussed by remember { mutableStateOf(false) }
   val expandedPeer = viewModel.expandedMenuPeer.collectAsState()
   val localClipboardManager = LocalClipboardManager.current
   val enableSearch = !isAndroidTV()
+  val enableNewSearch = false // This enables new search page with suggestions.
```

Comment:
Let's add a featureFlags.kt file and class and have feature flag constants available to all parts of the code instead of embedding this where it's used (to match the other code bases)...   That'll come in handy in the future.

Something like (sorry for the pseudo-code, I'm in swift mode):
```
class FeatureFlags {
   static const enabled_full_page_search: bool = false
}

...
if (FeatureFlags.enable_full_page_search) {

}
```